sql/database.sql

```diff
@@ -1430,6 +1430,7 @@ CREATE TABLE `drug_inventory` (
 
 DROP TABLE IF EXISTS `drug_sales`;
 CREATE TABLE `drug_sales` (
+  `uuid` binary(16) DEFAULT NULL COMMENT 'UUID for this drug sales record, for data exchange purposes',
```

Comment:
also need to add the uuid_unique key (already in the upgrade script)

Reply:
also missing the pharmacy_supply_type (already in the upgrade script)

Reply:
also missing 4 other new fields brought in via upgrade script

Reply:
Thanks for the catch.  Fixed this issue.